tracee-rules/metrics/stats.go

```diff
@@ -0,0 +1,9 @@
+package metrics
+
+import "github.com/aquasecurity/tracee/pkg/counter"
+
+type Stats struct {
```

Comment:
Since I had to split them from the common package and then merge the two packages (twice, in rules and ebpf), I think it just stayed that way in all that moving around. We could call it `metrics` now but the same thing was called `stats` before so would that be a blocker?

Reply:
I don't think it's a blocker. Just a reminder to stick to well defined terms and data models whenever possible. Especially that Prometheus or OpenTelemetry have well defined domains documented on https://opentelemetry.io/docs/concepts/data-sources/ and https://prometheus.io/docs/concepts/data_model/